include/circt/Support/LLVM.h

```diff
@@ -71,13 +71,15 @@ using mlir::TypeSwitch;
 // Forward declarations of LLVM classes to be imported in to the circt
 // namespace.
 namespace llvm {
+class BitVector;
 template <typename KeyT, typename ValueT, unsigned InlineBuckets,
           typename KeyInfoT, typename BucketT>
 class SmallDenseMap;
 } // namespace llvm
 
 // Import things we want into our namespace.
 namespace circt {
+using llvm::BitVector;
```

Comment:
Is this necessary given the imports in `FIRRTLAttributes.h`? 

I don't know what warrants inclusion in `LLVM.h`, but the usage of `BitVector` seems pretty narrow with this PR.

Reply:
When you put it that way, I do see this as being uncontroversial. Feel free to keep it in if that makes sense to you.